### ISSUE TITLE
Add Bungeecord explanation

### ISSUE THREAD

**Issue by Tzky** [SpongePowered/SpongeDocs#258]
As mentioned by @progwml6 on SoS4:
-add a bungeecord page
-explain what bungeecord is
-compatibility with sponge
-link to bungeecord page
-mention bungeecord offlinemode (warning?)

**Comment by kashike** [comment]
https://github.com/SpongePowered/SpongeCommon/issues/65

**Comment by Tzky** [comment]
To sum things up:
BungeeCord support won't happen soon, as there are technical issues with passing UUIDs to the offline server and players could possibly directly connect without using the bungeecord server.

**Comment by Tzky** [comment]
To sum it up again:
Blood has just pushed Bungeecord support for Sponge in SpongeCommon (see linked issue above). 
Time to document this, then. :)

**Comment by dualspiral** [comment]
Just to add, as I worked with @bloodmc on this - Sponge (but not Forge on it's own) will properly support IP/UUID forwarding with modded clients and as part of a server network **when the following conditions are true**:

* A modification to BungeeCord was required for this to work. [I am waiting on this PR](https://github.com/SpigotMC/BungeeCord/pull/1557) to BungeeCord to go through. I do have a build available in the commit message of my commit to Sponge(Forge), but I would rather that wasn't put into the docs, as we really should point people at official builds, and I plan to remove that build from my web server as soon as the PR is accepted.
* I also have a Spigot PR awaiting review that will allow FML clients to connect to Spigot servers through Bungee, and also be able to connect to modded Sponge servers. While this isn't strictly necessary for Sponge, for those who want to run Spigot servers alongside Sponge servers, it's pretty important!

**Comment by dualspiral** [comment]
I've written a quick gist with some information about where we are with BungeeCord, and what the state of play is:

https://gist.github.com/dualspiral/e13d91502fdef7c7ddd4

**Comment by Tzky** [comment]
Looks good and is by far more than i expected. Thanks for writing that up. I'll see that i open a PR on the Docs with your gist (md) converted to ``reST``.

**Comment by Tzky** [comment]
Closing this in favor of the linked PR #401.
